homeassistant/components/ollama/conversation.py

```diff
@@ -256,6 +257,7 @@ async def _async_handle_message(
                     # keep_alive requires specifying unit. In this case, seconds
                     keep_alive=f"{settings.get(CONF_KEEP_ALIVE, DEFAULT_KEEP_ALIVE)}s",
                     options={CONF_NUM_CTX: settings.get(CONF_NUM_CTX, DEFAULT_NUM_CTX)},
+                    think=settings.get(CONF_THINK),
```

Comment:
Does it work to update this to fall back to `DEFAULT_THINK` when not set so that it defaults off? otherwise users need to reconfigure or delete/re-add the integration for this to take effect. If this is supported, can you make this change?